src/start.ts

```diff
@@ -35,7 +35,12 @@ export const handler = async (_event: any = {}): Promise<any> => {
   });
   node.logger.info(`Airkeeper started at ${node.utils.formatDateTime(startedAt)}`, baseLogOptions);
 
-  const {  airnode: airnodeAddress, airnodeXpub: airnodeXpub ,chains: keeperChains, triggers: keeperTriggers } = keeperConfig;
+  const {
+    airnode: airnodeAddress,
+    airnodeXpub: airnodeXpub,
```

Comment:
Hey, well done on doing this. I think you can remove the `: airnodeXpub`, so it's just 
```
const {  airnode: airnodeAddress, airnodeXpub,chains: keeperChains, triggers: keeperTriggers } = keeperConfig;
```
It's generally better to not map variable names if you don't need to, so if there's a way of keeping airnode = airnode, chains = chain, etc. then you can just do 
```
const {  airnode, airnodeXpub, chains, triggers } = keeperConfig;
```
which is easier to read/less confusing :)

Reply:
thanks for the suggestion, The xpub is needed to derive the sponsorWallets so I think I have to keep it there.

> It's generally better to not map variable names if you don't need to, so if there's a way of keeping airnode = airnode, chains = chain, etc. then you can just do

ah okay, well I didn't want to disturb santiago's code as much as possible but i'll do the other suggestion.

Reply:
If you're busy with the conference I can make the changes? This would be useful for the backup Amberdata Airkeeper.

Let me know. I'll probably still need assistance with the sponsor wallets.

Reply:
Yeah go ahead, im travelling today so won't be able to get to it until in the evening